script/lib/transpile-packages-with-custom-transpiler-paths.js

```diff
@@ -1,29 +1,36 @@
 'use strict'
 
 const CompileCache = require('../../src/compile-cache')
-const fs = require('fs')
+const fs = require('fs-extra')
 const glob = require('glob')
 const path = require('path')
 
 const CONFIG = require('../config')
+const backupNodeModules = require('./backup-node-modules')
+const runApmInstall = require('./run-apm-install')
+
+require('colors')
```

Comment:
Although I agree with you in principle, `color` is already used elsewhere in the Atom repo, while `chalk` would have involved bringing in a new dependency.